examples/api/custom_projection_example.py

```diff
@@ -291,12 +291,12 @@ class DegreeFormatter(Formatter):
         """
 
         def __init__(self, round_to=1.0):
+            super(HammerAxes.DegreeFormatter, self).__init__()
             self._round_to = round_to
 
-        def __call__(self, x, pos=None):
+        def format_for_tick(self, x, pos=None):
```

Comment:
Again a much needed improvement IMHO. If we really want `__call__()` functionality I would just call `format_for_tick()` without any additional code.